issues/19/readme.md

```diff
@@ -0,0 +1,31 @@
+# The issue
```

Comment:
What is the meaning of this file, I never see something like this.

Reply:
Nevermind, it's just me:
sometimes i keep important issues in a folder called issue, because it contains useful information that can be consulted when needed.
I'll remove it.👍🏼